ozaria/site/components/teacher-dashboard/Panel/components/ConceptCheckInfo.vue

```diff
@@ -36,8 +41,15 @@ export default {
     >
       <img src="/images/ozaria/teachers/dashboard/svg_icons/Icon_TimeSpent.svg">
       <p>
-        <b>{{ 'Time Spent: ' }}</b>
-        {{ `${conceptCheck.timeSpent} min` }}
+        <b>{{ $t('teacher.time_played_label') }}</b>
+        {{ `${conceptCheck.timeSpent}` }}
```

Comment:
if we add unit `min` in threshold, we should add unit here too. also, maybe we can add some green/red color for timeSpent which shorter/longer than the threshold for easy checking